Cleanup open-source comment

Potherca opened the pull request:
Ahoy!

When I looked at the page-source of the Corona Check website, I noticed the open-source comment:

```html
<!--
  This website is generated at 2021-06-24 11:21:14 +0000.
  The code is open source. You are welcome to contribute. Go to git: 
  https://github.com/minvws/nl-covid19-coronacheck-website
-->
```

So, being curious,  I visited GitHub and had a look at where the comment comes from.

In the code I noticed that the comment was not [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) but repeated across the various Jekyll layout files.

This merge request resolves that by placing the comment in a separate include file.

Besides that, these changes are also made:

- The placement of the comment has been made consistent across layouts (always after the `<html>` tag)
- The white-space around the comment has been made consistent across layouts
- The wording has been changed from `Go to git: ` to `Go to:`, as _technically_ the link does not go to Git but GitHub. (And changing it to `Go to GitHub:` seemed needlessly verbose).

The comment **has not** been added to [`print-portal/public/index.html`](https://github.com/minvws/nl-covid19-coronacheck-website/blob/main/print-portal/public/index.html) as I was not certain that content was also meant to be parsed by Jekyll (both now and in the future).

The result can be seen functioning at: https://contrib.pother.ca/nl-covid19-coronacheck-website/

anneke commented:
Hi @Potherca , thank you for helping out! I think we have the same happening over at 

https://github.com/minvws/nl-covid19-notification-app-website 

if you'd like to help there, too. I see that because of everything that happened in a private copy of this repo (due to all the politically sensitive information), this branch is a little outdated. If you give me access rights to your fork I can help you fix it! Let me know :) (preferably on twitter ;) )

Potherca commented:
Thanks for the offer! I've updated `main` on my fork and rebased this branch.
Everything _should_ be up to date now, let me know if I missed something.

I'll have a look over at the app website :+1:

anneke reviewed (approved):
Much better this! Thanks again Ben!
